mod/robots_txt.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @file mod/robots_text.php
+ * @brief Module which returns the default robots.txt
+ * @version 0.1.2
+ */
+
+
+/**
+ * @brief Return default robots.txt when init
+ * @param App $a
+ * @return void
+ */
+function robots_txt_init(App $a)
+{
+
+	/** @var string[] globally disallowed url */
```

Comment:
Just on last little thing. Is this also phpdoc style? All comments with `/**` at the beginning will be inserted into the doxygen docu.